dpctl/tests/test_sycl_device.py

```diff
@@ -163,13 +163,12 @@ def test_equal():
     "atomic64",
     "usm_atomic_host_allocations",
     "usm_atomic_shared_allocations",
+    "emulated",
 ]
 
 # SYCL 2020 spec aspects not presently
 # supported in DPC++, and dpctl
-list_of_unsupported_aspects = [
-    "emulated",
-]
+list_of_unsupported_aspects = []
```

Comment:
@oleksandr-pavlyk 
Should this list be removed at this point?

Reply:
This list feeds into a fixture that parametrizes test for unsupported aspects. I'd keep it in case spec adds an aspect which compiler does not yet support.